executive/Q5015503/current/popolo-m17n.json

```diff
@@ -333,54 +124,16 @@
       }
     },
     {
-      "id": "http://www.wikidata.org/entity/statement/Q312361-61d4c7dd-4041-f928-5245-64fd983a9e34",
-      "person_id": "Q312361",
-      "on_behalf_of_id": "Q2745857",
-      "organization_id": "Q5015503",
-      "area_id": "Q155",
-      "role_superclass_code": "Q5176750",
-      "role_superclass": {
-        "lang:pt-br": "Presidente do Brasil",
-        "lang:pt": "Presidente do Brasil",
-        "lang:en": "President of Brazil"
-      },
-      "role_code": "Q5176750",
-      "role": {
-        "lang:pt-br": "Presidente do Brasil",
-        "lang:pt": "Presidente do Brasil",
-        "lang:en": "President of Brazil"
-      }
-    },
-    {
-      "id": "http://www.wikidata.org/entity/statement/Q314698-ced5ffe6-4eab-10eb-9929-0f42f6ca27cf",
-      "person_id": "Q314698",
-      "on_behalf_of_id": "Q7314669",
-      "organization_id": "Q5015503",
-      "area_id": "Q155",
-      "role_superclass_code": "Q5176750",
-      "role_superclass": {
-        "lang:pt-br": "Presidente do Brasil",
-        "lang:pt": "Presidente do Brasil",
-        "lang:en": "President of Brazil"
-      },
-      "role_code": "Q5176750",
-      "role": {
-        "lang:pt-br": "Presidente do Brasil",
-        "lang:pt": "Presidente do Brasil",
-        "lang:en": "President of Brazil"
-      }
-    },
-    {
-      "id": "http://www.wikidata.org/entity/statement/Q314702-d3df0bed-487f-e607-50fd-bd82ea5f16b0",
-      "person_id": "Q314702",
-      "on_behalf_of_id": "Q834073",
+      "id": "http://www.wikidata.org/entity/statement/Q273264-15AC7416-699F-4937-B12D-6F1743DBAE0F",
+      "person_id": "Q273264",
+      "on_behalf_of_id": "Q1322751",
       "organization_id": "Q5015503",
       "area_id": "Q155",
-      "role_superclass_code": "Q5176750",
+      "role_superclass_code": "Q2285706",
```

Comment:
We really wanted this to be president, but the President of Brazil is both HoS and HoG, and so has president and HoG as superclasses, and we can't (I think) reliably pick between them.